modules/test_user_dashboard/app/helpers/test_user_dashboard/tud_accounts_helper.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module TestUserDashboard
+  module TudAccountsHelper
```

Comment:
Is this just a placeholder file for now?

Reply:
It was autogenerated when the `TudAccounts` resource was created, should it be removed since it's just a placeholder?

Reply:
I would say yes. I took a glance at existing modules and it looks like they don't have a helpers dir/placeholder files